pkg/vault/test/Authorizer.test.ts

```diff
@@ -87,30 +131,62 @@ describe('Authorizer', () => {
         authorizer = authorizer.connect(admin);
       });
 
-      context('when the roles where granted', () => {
+      context('when the roles ANYWHERE granted to a set of contracts', () => {
```

Comment:
```suggestion
      context('when the roles were granted to a set of contracts', () => {
```

We had a typo in the original description ('where' instead of 'were'), and it seems that fell prey to your find+replace :)